Automattic/jetpack
# Add/protect waiting message

#### Changes proposed in this Pull Request:

* Changes the wording for the initial screen.

#### Jetpack product discussion


1201069996155224-as-1202326318206097

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Checkout this branch.
* Open the Protect page under the Jetpack menu.
* Verify that the wording was changed from `This could take a few seconds` to `This could take a minute or two.`
